media/css/products/vpn/common.scss

```diff
@@ -56,10 +91,11 @@ $image-path: '/media/protocol/img';
 .guarantee-copy {
     color: $color-vpn-brand-violet;
     font-weight: bold;
-    margin-bottom: $spacing-xl;
+    margin-bottom: $spacing-lg;
 
     &:before {
         @include bidi((
+            (background-position, top left, top right),
```

Comment:
There's an [extra reference of `.t-dark`](https://github.com/alexgibson/bedrock/blob/vpn-variable-pricing/media/css/products/vpn/common.scss#L111-L118) in this file.

Reply:
Good catch!